projects/components/src/lib/directives/index.ts

```diff
@@ -9,3 +9,6 @@ export * from './responsive-input/responsive-input.directive';
 export * from './responsive-input/responsive-input.module';
 export * from './show-clipped-text.directive';
 export * from './show-clipped-text.directive.module';
+export * from './aria-activedescendant/aria-activedescendant.directive';
```

Comment:
I now it hasn't been done everywhere but it's better practice to create an `index.ts` in the subfolder exporting that folder's contents and then doing

```
export * from './aria-activedescendant`
```